### PR TITLE
DevKit updates for 5.x branch

### PR DESCRIPTION
**Pull request by SonataCI**
## Changelog

```markdown
### Removed
- Support for PHP 7.4
- Support for Symfony 6.0 and 6.1
```

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Any idea why the existing `@psalm-suppress UndefinedDocblockClass` doesn't work anymore @jordisala1991 ?

**Comment by jordisala1991** [comment]
I dont understand why

**Comment by VincentLanglet** [comment]
Baseline works, so let's go with this @jordisala1991

File: src/Document/UserManager.php [hunk 1]
```diff
@@ -28,38 +28,24 @@
...
         string $class,
         ManagerRegistry $registry,
-        CanonicalFieldsUpdaterInterface $canonicalFieldsUpdater,
-        // @phpstan-ignore-next-line
-        object $userPasswordHasher
+        private CanonicalFieldsUpdaterInterface $canonicalFieldsUpdater,
+        /**
+         * @phpstan-ignore-next-line
+         * @psalm-suppress UndefinedDocblockClass
+         */
```
Comment (jordisala1991) [inline src/Document/UserManager.php]:
```suggestion
        // @phpstan-ignore-next-line
```

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**